reporting_app/static/datatable.js

```diff
@@ -277,14 +280,13 @@ var configure_dt = function(dt_config) {
         'searching': dt_config.searching,
         'info': dt_config.info,
         'processing': true,
-        'serverSide': serverSide,
+        'serverSide': false,
```

Comment:
Have we considered whether it will scale, since it is no longer server side?

Reply:
We've never used server side processing in production.
`severSide` was set up in the previous PR but I realised it was not necessary.